### ISSUE TITLE
Possibility to inject raw value ?

### ISSUE THREAD

**Issue by ektarum** [ComPHPPuebla/dbal-fixtures#7]
Hi,

I attempt to insert this raw value : AsText(POINT(longitude latitude)) 
where longitude and latitude are floats.

It is possible in any manner to inject raw string ? Actually is quoted by dbal connection because no types has passed.

Faker provider no change this because the quotes are add after.

Thank's in advance.

**Comment by MontealegreLuis** [comment]
Hi @ektarum. Is this MySQL? Could you please share with my the entry in your `yml` file that produces this error? And your `CREATE TABLE` statement please? 


Thank you!

**Comment by ektarum** [comment]
Hi @MontealegreLuis , sorry for late response.

i shared a sample with you :

creation of table :
```sql
CREATE TABLE coord_test (
  id int unsigned NOT NULL AUTO_INCREMENT,
  coordinates point DEFAULT NULL,
  PRIMARY KEY (id)
) ENGINE=InnoDB;
```

SQL insert example :
```sql
INSERT INTO coord_test VALUES (null, PointFromText('POINT(51.8939035 4.5231352)'));
```

sample yaml:
```yaml
coord_test:
    coord_test_1:
        id: 12
        coordinates: PointFromText('POINT(51.8939035 4.5231352)')
```

the problem is the transformation of ```PointFromText('POINT(51.8939035 4.5231352)')``` to ```"PointFromText('POINT(51.8939035 4.5231352)')"``` (quote adding).


for doctrine i've declare a type, this types is inspired by this : https://codeutopia.net/blog/2011/02/19/using-spatial-data-in-doctrine-2/

Thank you.

**Comment by MontealegreLuis** [comment]
Thank you @ektarum I'll take a look later today!

**Comment by MontealegreLuis** [comment]
Hi @ektarum 

I've pushed the v4.1 branch with a fix for this. Please take a look and test it. If it solves your issue I'll release 4.1.

```
composer require comphppuebla/dbal-fixtures 4.1.x-dev
```

I had to create the `insert` programmatically in order to be able to use values like the ones described in this issue.

I added documentation for the new syntax to be used for this type of values here.

https://github.com/ComPHPPuebla/dbal-fixtures/blob/v4.1.0/docs/sql-functions.md

Please let me know if it's unclear or if you need help.

I also added documentation to explain how to register specific platform types like `POINT`

https://github.com/ComPHPPuebla/dbal-fixtures/blob/v4.1.0/docs/platform-types.md

Thank you!

**Comment by MontealegreLuis** [comment]
Excellent @ektarum! I just released [4.1.0](https://github.com/ComPHPPuebla/dbal-fixtures/releases/tag/4.1.0) with this feature.

Thank you!
